src/annotator/test/annotation-sync-test.js

```diff
@@ -1,21 +1,19 @@
-import EventEmitter from 'tiny-emitter';
+import { EventBus } from '../util/emitter';
 
 import AnnotationSync from '../annotation-sync';
 
 describe('AnnotationSync', () => {
-  let createAnnotationSync;
+  let emitter;
   let fakeBridge;
-  let options;
+  let createAnnotationSync;
   let publish;
```

Comment:
Can you keep these variables sorted. Its easier to scan them and insert a new entry if they are.

Reply:
I sorted in the order in which they are assigned in the `beforeEach`. I will sort them alphabetically.